R/plot-timeprofile.R

```diff
@@ -257,6 +257,7 @@ plotTimeProfile <- function(data = NULL,
     getAestheticValues(
       n = totalLength - colorLength,
       selectionKey = plotConfiguration$points$color,
+      position = colorLength,
       aesthetic = "color"

```

Comment:
Setting a position to a number of colors is not obvious. This needs to be commented